editor/editor_properties.h

```diff
@@ -91,16 +91,40 @@ class EditorPropertyMultilineText : public EditorProperty {
 
 class EditorPropertyTextEnum : public EditorProperty {
 	GDCLASS(EditorPropertyTextEnum, EditorProperty);
-	OptionButton *options;
 
+	HBoxContainer *default_layout;
+	HBoxContainer *edit_custom_layout;
+
+	OptionButton *option_button;
+	Button *edit_button;
+
+	LineEdit *custom_value_edit;
+	Button *accept_button;
+	Button *cancel_button;
+
+	Vector<String> options;
+	bool string_name = false;
+	bool loose_mode = false;
+
+	enum MenuOption {
+		EDIT_VALUE,
+		CLEAR_VALUE,
+	};
```

Comment:
Seems unused.

Reply:
Ah yes, forgot to remove after some experimenting, good catch.